js/wizard.js

```diff
@@ -0,0 +1,437 @@
+function validateIPaddress(ipaddress) {
+  if (/^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/.test(ipaddress)) {
+    return (true);
+  }
+  return (false);
+}
+
+function validateFQDN(fqdn)
+{
+  if (/^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/.test(fqdn)) {
+    return (true);
+  }  return (false);
+
+}
+
+function validateNetmask(netmask)
+{
+  if (/^(254|252|248|240|224|192|128)\.0\.0\.0|255\.(254|252|248|240|224|192|128|0)\.0\.0|255\.255\.(254|252|248|240|224|192|128|0)\.0|255\.255\.255\.(254|252|248|240|224|192|128|0)$/.test(netmask)) {
+    return (true);
+  }
+  return (false);
+}
+
+function convertNetmaskToCIDR(netmask) {
+  var maskNodes = netmask.match(/(\d+)/g);
+  var cidr = 0;
+  for(var i in maskNodes)
+  {
+    cidr += (((maskNodes[i] >>> 0).toString(2)).match(/1/g) || []).length;
+  }
+  return cidr;
+}
+
+
+function saveFile(filename, data) {
+  var blob = new Blob([data], {type: 'text/csv'});
```

Comment:
Are we assuming IE11 and above? If not, would be good to preform at least a **try/catch**.